dataactcore/utils/jobQueue.py

```diff
@@ -34,7 +34,7 @@ def enqueue(jobID):
             response = requests.post(url, params)
             return response.json()
 
-        @self.jobQueue.task(name='jobQueue.generate_d1')
+        @self.jobQueue.task(name='jobQueue.generate_d_file')
```

Comment:
@nmonga91 I don't know enough about how the jobQueue works to understand if this change impacts validation functionality or if it's just a change to improve the the task name. Can you clarify? Thanks!